ledger-tool/src/main.rs

```diff
@@ -686,7 +695,14 @@ fn open_blockstore(
 }
 
 fn open_database(ledger_path: &Path, access_type: AccessType) -> Database {
-    match Database::open(&ledger_path.join("rocksdb"), access_type, None) {
+    match Database::open(
+        &ledger_path.join("rocksdb"),
+        BlockstoreOptions {
+            access_type,
+            recovery_mode: None,
+            ..BlockstoreOptions::default()
+        },
```

Comment:
I was initially a little undecided on use of the `..BlockstoreOptions::default()` to auto-populate rest of the struct; the tradeoff is obviously convenience (ie callers auto-update) vs. having tailored values for each caller. But, our defaults should be good so I think I'm good with this as-is; no changes requested, just some rambling